packages/manager/apps/cloud/client/app/dbaas/logs/detail/index/add/logs-index-add.controller.js

```diff
@@ -3,23 +3,25 @@ class LogsIndexAddModalCtrl {
     $q,
     $stateParams,
     $uibModalInstance,
+    $translate,
     CucControllerHelper,
     indexInfo,
-    options,
-    quota,
     LogsIndexService,
+    LogsConstants,
   ) {
     this.$stateParams = $stateParams;
     this.$q = $q;
     this.CucControllerHelper = CucControllerHelper;
     this.indexInfo = indexInfo;
-    this.options = options;
-    this.quota = quota;
     this.suffixPattern = '^[a-z0-9_-]+$';
     this.LogsIndexService = LogsIndexService;
+    this.LogsConstants = LogsConstants;
     this.$uibModalInstance = $uibModalInstance;
+    this.$translate = $translate;
     this.serviceName = $stateParams.serviceName;
     this.index = this.LogsIndexService.getNewIndex();
+    this.indexPrice = { shard: { price: '' }, volume: { price: '' } };
+    this.initLoaders();
```

Comment:
Please set this call into $onInit, it is disturbing to have this call in the constructor, and the $onInit code called in parallel. Or maybe there is a reason to have it there ?

Reply:
IMHO no more than this https://github.com/ovh/manager/blob/15584f27719dc4adc374b790ad5d2506888b8da8/packages/manager/apps/cloud/client/app/dbaas/logs/detail/index/add/logs-index-add.controller.js#L16

Reply:
You're right, neither this should be into constructor

Reply:
Can't find blocker of using `constructor` vs `$onInit` hooks.

Both usage are correct but I admit that for consistency, we should work to uniformized them.

Not mandatory for the enhancements requested by @pdepaepe!